? JetMETCorrections/METPUSubtraction/data/gbrmet_53_Dec2012.root
? JetMETCorrections/METPUSubtraction/data/gbrmet_53_Dec2012_UnityResponse.root
? JetMETCorrections/METPUSubtraction/data/gbrmet_53_Dec2012_highqt.root
? JetMETCorrections/METPUSubtraction/data/gbrmet_53_Dec2012_highqt_UnityResponse.root
? JetMETCorrections/METPUSubtraction/data/gbrmetphi_53_Dec2012.root
? JetMETCorrections/METPUSubtraction/data/gbru1cov_53_Dec2012.root
? JetMETCorrections/METPUSubtraction/data/gbru1cov_53_Dec2012_UnityResponse.root
? JetMETCorrections/METPUSubtraction/data/gbru1cov_53_Dec2012_highqt.root
? JetMETCorrections/METPUSubtraction/data/gbru2cov_53_Dec2012.root
? JetMETCorrections/METPUSubtraction/data/gbru2cov_53_Dec2012_UnityResponse.root
? JetMETCorrections/METPUSubtraction/data/gbru2cov_53_Dec2012_highqt.root
? JetMETCorrections/METPUSubtraction/python/__init__.py
Index: JetMETCorrections/METPUSubtraction/src/PFMETAlgorithmMVA.cc
===================================================================
RCS file: /local/reps/CMSSW/CMSSW/JetMETCorrections/METPUSubtraction/src/PFMETAlgorithmMVA.cc,v
retrieving revision 1.7
diff -w -r1.7 PFMETAlgorithmMVA.cc
66,69d65
< <<<<<<< PFMETAlgorithmMVA.cc
<   //is42_ = ( inputFileNameU.fullPath().find("42") != std::string::npos ) ?
<   //  true : false;
< =======
71d66
< >>>>>>> 1.6
147,150d141
< <<<<<<< PFMETAlgorithmMVA.cc
<   if(isOld42_) ptThreshold = 1.;  //PH: For 42 training added a pT cut of 1 GeV on corrected Jets
<   std::vector<mvaMEtUtilities::JetInfo>          jets_cleaned         = utils_.cleanJets(jets, leptons, ptThreshold, 0.5);
< =======
153d143
< >>>>>>> 1.6
Index: JetMETCorrections/METPUSubtraction/test/setup.sh
===================================================================
RCS file: /local/reps/CMSSW/CMSSW/JetMETCorrections/METPUSubtraction/test/setup.sh,v
retrieving revision 1.7
diff -w -r1.7 setup.sh
23d22
< cvs co -r V01-05-07                             RecoTauTag/RecoTau 
